src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -166,20 +166,71 @@
          </widget>
         </item>
         <item>
-         <widget class="QCheckBox" name="userspaceCheckBox">
-          <property name="visible">
-           <bool>true</bool>
-          </property>
+         <widget class="QCheckBox" name="introspectionCheckBox">
           <property name="text">
-           <string>Enable user space instrumentation</string>
+           <string>Enable introspection</string>
           </property>
          </widget>
         </item>
         <item>
-         <widget class="QCheckBox" name="introspectionCheckBox">
-          <property name="text">
-           <string>Enable introspection</string>
+         <widget class="QWidget" name="dynamicInstrumentationMethodWidget" native="true">
+          <property name="toolTip">
+           <string>Method 'Orbit' has lower overhead but is still experimental.</string>
           </property>
+          <layout class="QHBoxLayout" name="horizontalLayout_3">
+           <property name="leftMargin">
+            <number>0</number>
+           </property>
+           <property name="topMargin">
+            <number>0</number>
+           </property>
+           <property name="rightMargin">
+            <number>0</number>
+           </property>
+           <property name="bottomMargin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QLabel" name="label_2">
+             <property name="text">
+              <string>Dynamic Instrumentation Method:</string>
```

Comment:
```suggestion
              <string>Dynamic instrumentation method:</string>
```
Compare with the other options, that are not capitalized. E.g., "Callstack unwinding method".

Reply:
done